### DISCUSSION TITLE
More GUI's

### DISCUSSION THREAD

**Discussion by winnipeg21** [Zrips/CMI#4822] [Feature Requests]
So we have /craft. Why not add /furance, /anvil, /smithingtable (/st), /enchanter, etc.. Could be useful for perks and admin use too.

**Comment by mrfloris** [comment]
Yeah that would be nice, I think more have -s now, but a few got overlooked, or there's a particular reason they cant or wont have the -s option.

**Comment by winnipeg21** [comment]
Man created this over a year ago. Ain't judging do feel there needs to be more done to compete with essentials.

Personally moved away from CMI there's just too many niche features and not enough to complete replace essentials. Like /furnace, /anvil, etc. It's a great plugin don't get me wrong but if you don't need those niche features you are better off with the alternatives. 

Main thing that moved me was rgb formating wasn't compatible with my tab and chat formating. Made it more difficult. 

Also I feel a whole locale overhaul is needed to make the text less "professional" and look modern. But that's another suggestion/project someone can do.
